www-plugins/passff-host/passff-host-1.0.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_4 )
```

Comment:
No support for newer Python versions? 3.5 is the bare minimum these days, and 3.6 is coming in big steps.